### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by frankiefu**

### CODE REVIEW

File: test/shop-cart.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        // Data bindings will stamp out new DOM asynchronously
+        // so wait to check for updates
+        flush(function() {
+          var quantity = cart.shadowRoot.querySelector('header').querySelector('span');
+          assert.equal('(' + cart._getPluralizedQuantity(cartData.length) + ')', quantity.textContent, 'quantity should render correctly');
+          done();
+        });
+      });
+
+      test('main section should have visible attirubte when cart has items', function(done) {
```
Comment (keanulee) [inline test/shop-cart.html]: `attribute`, here and everywhere

File: test/shop-cart.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+          quantity: 1,
+          size: "M"
+        }
+      ]
+
+      setup(function() {
+        cart = fixture('basic');
+      });
+
+      test('cart item lengths should be equalled', function(done) {
```
Comment (keanulee) [inline test/shop-cart.html]: `should be equal`, here and everywhere

File: test/shop-cart.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        // Data bindings will stamp out new DOM asynchronously
+        // so wait to check for updates
+        flush(function() {
+          var section = cart.shadowRoot.querySelectorAll('.subsection')[1];
+          assert.isFalse(section.hasAttribute('visible'), 'main section should not have visible attirubte when cart is empty');
+          done();
+        });
+      });
+
+      test('change-section event should fire', function(done) {
```
Comment (keanulee) [inline test/shop-cart.html]: 👍  for event API tests

**Comment by frankiefu** [comment]
@keanulee Updated. PTAL.

**Pull request merged**